packages/runtime-core/src/components/Teleport.ts

```diff
@@ -133,7 +133,11 @@ export const TeleportImpl = {
       if (disabled) {
         mount(container, mainAnchor)
       } else if (target) {
-        mount(target, targetAnchor)
+        parentSuspense
```

Comment:
Why not use `queuePostRenderEffect`?
I feel it should be:
```ts
  if(parentSuspense){
    queuePostRenderEffect(()=>{mount(target, targetAnchor)})
  }
```
(**NOT tested**)

Reply:
cool! I get it